src/d3d/nvapi_d3d_instance.h

```diff
@@ -10,15 +10,15 @@ namespace dxvk {
         ~NvapiD3dInstance();
 
         void Initialize();
-        [[nodiscard]] bool IsReflexAvailable();
-        [[nodiscard]] bool IsReflexEnabled() const;
-        void SetReflexEnabled(bool value);
-        void Sleep();
-        void SetTargetFrameTime(uint64_t frameTimeUs);
+        [[nodiscard]] bool IsReflexAvailable(IUnknown* device);
+        [[nodiscard]] bool IsLowLatencyEnabled() const;
+        [[nodiscard]] bool IsUsingLfx() const;
+        void SetReflexMode(IUnknown* device, bool enable, bool boost, uint64_t frameTimeUs);
```

Comment:
We should change this declaration (and its implementation) to `[[nodiscard]] bool` since `NvapiD3dLowLatencyDevice::SetLatencySleepMode` returns a boolean indicating success. The same applies to `Sleep` below.